latch/functions/secrets.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from latch_cli.config.latch import _LatchConfig
+from latch_cli.tinyrequests import post
+from latch_cli.utils import current_workspace, retrieve_or_login
+
+config = _LatchConfig()
+endpoints = config.sdk_endpoints
+
+
+def get_secret(secret_name: str):
```

Comment:
shouldn't this be in the SDK part of latch and not in CLI?

Reply:
its not in the cli?